### PR TITLE
Get tests passing on Python 3

### PR DESCRIPTION
**Pull request by millerdev**
There is a lot more modernization work that still needs to be done, but this branch is a minimal change set to get tests running on Python 3. Hopefully this will be possible without tackling much of the unicode/bytes work, which is still pending.

### CODE REVIEW

**Comment by dannyroberts** [comment]
In case it saves you some time, I can reproduce the latest travis test failure locally with `manage-commcare-cloud make-docs > docs/commcare-cloud/commands/index.md` That's called by `make`; `make` caches, so try `make clean; make` if you want to reproduce it that way and test the rest of that. There is also, separately, `make requirements` which runs during tests to make sure your requirements.txt is consistent with your `setup.py`. (Python2 and python3 might need separate requirements files.)

**Comment by millerdev** [comment]
@dannyroberts Yes, thanks. I'm also able to reproduce locally. Some of the errors I was hitting earlier were more specific to differences between the travis build environment and mine and were harder to reproduce.

**Comment by millerdev** [comment]
@dannyroberts @gherceg tests passed. :white_check_mark: This is ready for review.

File: src/commcare_cloud/environment/schemas/postgresql.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 PostgresqlOverride = get_defaults_jsonobject(
     'postgresql_base',
-    allow_dump_from_pgstandby=jsonobject.BooleanProperty,
+    allow_dump_from_pgstandby=jsonobject.BooleanProperty(),
```
Comment (dannyroberts) [inline src/commcare_cloud/environment/schemas/postgresql.py:18]: Interesting. Your change looks right to me

**Review by dannyroberts — APPROVED** [review]
This is super awesome!

**Review by gherceg — APPROVED** [review]

**Pull request merged**